### PR TITLE
Update to PMD 6.4.0

### PR DESCRIPTION
**Pull request by mrfyda**

### CODE REVIEW

File: src/main/resources/docs/description/description.json [hunk 1]
```diff
@@ -101,13 +101,6 @@
 }, {
   "patternId" : "category_java_errorprone_UselessOperationOnImmutable",
   "title" : "An operation on an Immutable object (String, BigDecimal or BigInteger) won't change the object itself"
-}, {
```
Comment (mrfyda) [inline src/main/resources/docs/description/description.json]: The DocGenerator should sort this file by `patternId` so we get stable diffs.

Comment (rtfpessoa) [reply src/main/resources/docs/description/description.json]: that would be nice

File: build.sbt [hunk 2]
```diff
@@ -75,7 +75,7 @@ daemonUser in Docker := dockerUser
 
 daemonGroup in Docker := dockerGroup
 
-dockerBaseImage := "develar/java"
```
Comment (rtfpessoa) [inline build.sbt]: The tests seem to be failing due to this

**Comment by rsoesemann** [comment]
Thanks to all for updating!

**Pull request merged**